frontend/views/containers/sidebar/GroupsMinIncome.vue

```diff
@@ -10,27 +10,52 @@ div
     :aria-label='L("Change Mincome")'
     @click='openProposal'
   ) Change
+  br
+  i18n.link(
+    tag='button'
+    @click='changeMin'
+  ) Increase {{ currency }}10
 </template>
 
 <script>
 import sbp from '~/shared/sbp.js'
 import { OPEN_MODAL } from '@utils/events.js'
 import currencies from '@view-utils/currencies.js'
-import { mapGetters } from 'vuex'
+import { mapState, mapGetters } from 'vuex'
 
 export default {
   name: 'GroupsMinIncome',
   computed: {
     currency: function () {
       return currencies[this.groupSettings.incomeCurrency]
     },
+    ...mapState([
+      'currentGroupId'
+    ]),
     ...mapGetters([
       'groupSettings'
     ])
   },
   methods: {
     openProposal () {
       sbp('okTurtles.events/emit', OPEN_MODAL, 'MincomeProposal')
+    },
+    async changeMin () {
+      const updatedSettings = await sbp('gi.contracts/group/updateSettings/create',
+        {
+          incomeProvided: this.groupSettings.incomeProvided + 10
+        },
+        this.currentGroupId // NOTE 1: why do i need to pass currentGroupId?
+        // Actually, I understand why, to give context... but, how would I know it?
+        // When reading the selector at group.js, that's expecting state, data, meta,
+        // but not a groupId... It took me a **while** to realize that
+        // when I call this selector, it actually calls Contract.js 'selectors/register'
+        // first and only then it calls GIMessage.create() (what for?)
```

Comment:
Calling this selector does *not* call "Contract.js 'selectors/register'".

It's unclear to me what the question here is, but maybe a call would be better to go over this.

Reply:
Hum.... but I added a log there at line 28, right after `[`${action}/create`]:` function and it passes there when I call sbp create selector... so, that's why I assumed that calling 'create' selector, calls the register somehow...